pkg/cmd/gist/create/create.go

```diff
@@ -153,6 +153,12 @@ func createRun(opts *CreateOptions) error {
 			if httpError.OAuthScopes != "" && !strings.Contains(httpError.OAuthScopes, "gist") {
 				return fmt.Errorf("This command requires the 'gist' OAuth scope.\nPlease re-authenticate by doing `gh config set -h github.com oauth_token ''` and running the command again.")
 			}
+			if httpError.StatusCode == http.StatusUnprocessableEntity {
+				if detectEmptyFiles(files) {
+					fmt.Fprintf(errOut, "%s Failed to create gist: %s", cs.Red("X"), errors.New("A gist file must not contain only whitespace characters"))
```

Comment:
1. You don't need to construct an `errors.New()` just to pass it as an argument for the `%s` placeholder. A simple string will suffice
2. Let's use `cs.FailureIcon()` instead of `cs.Red()`
3. In case a completely empty file was passed, the current message might be a slight misnomer. What do you think about: `Failed to create gist: a gist file cannot be blank`